include/exadg/compressible_navier_stokes/spatial_discretization/kernels_and_operators.h

```diff
@@ -930,6 +931,7 @@ struct ViscousOperatorData
 
   unsigned int dof_index;
   unsigned int quad_index;
+  unsigned int quad_index_standard;
```

Comment:
If I remember correctly, we never introduced `Kernel` or `KernelData` for the module `CompNS`. So I agree with the design chosen here, namely adding the parameter to `OperatorData`.  For clarity, I think the name `quad_index_penalty_parameter` (or similar) would be better. It is probably not used for anything else.